doc/index.rst

```diff
@@ -386,3 +339,26 @@
     <script>
       $('#testimonials_carousel').carousel()
     </script>
+
+.. We want the previous/next button to work on the user guide pages and on the
+   API. We have to put the doctree so that sphinx populates the
+   rellinks. Yet, we don't want it to be displayed on the main page, hence we
+   use CSS to hide it.
+
+.. raw:: html
+
+   <div class="no-display">
```

Comment:
this is really an ugly hack. there are :hidden: toc-trees http://sphinx-doc.org/markup/toctree.html which is what we should use here